Assets/Scripts/Controllers/GroundController.cs

```diff
@@ -66,7 +66,7 @@ private void SpawnObstacles()
             var spawnCount = CalculateNumberOfObstaclesToSpawn();
             for (var i = 0; i < spawnCount; i++)
             {
-                _obstacleController.SpawnHazard(parent, _lastGroundWidth);
+                _obstacleController.SpawnHazard(parent, 10f, 4f);
```

Comment:
This is approximately the 30*12 land object dived by 3. I don't really know why there is a 1/3 scale applied...